docs/examples/hypervisors/kvm/network.tf

```diff
@@ -0,0 +1,132 @@
+resource "oci_core_virtual_network" "company-vcn" {
+  cidr_block     = "10.0.0.0/16"
+  compartment_id = "${var.compartment_ocid}"
+  display_name   = "${var.customer_name}-vcn"
+  dns_label      = "${var.customer_name}"
+}
+
+resource "oci_core_internet_gateway" "company-ig" {
+  compartment_id = "${var.compartment_ocid}"
+  display_name   = "${var.customer_name}-ig"
+  vcn_id         = "${oci_core_virtual_network.company-vcn.id}"
+}
+
+resource "oci_core_route_table" "backend-rt" {
+  compartment_id = "${var.compartment_ocid}"
+  vcn_id         = "${oci_core_virtual_network.company-vcn.id}"
+  display_name   = "${var.customer_name}-backend-rt"
+
+  route_rules {
+    cidr_block        = "0.0.0.0/0"
+    network_entity_id = "${oci_core_internet_gateway.company-ig.id}"
```

Comment:
Backend route table should use the private IP OCID of the kvm-guest-frontend-vnic.